apps/web/src/components/Example/Example.simple.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
+
+import Example from "./Example";
+
+// The default export metadata controls how Storybook lists your stories and provides information used by addons.
+const meta: Meta<typeof Example> = {
+  component: Example,
+  // You can also define args at the component level; they will apply to all the component's stories unless you overwrite them. To do so, use the args key on the default CSF export:
```

Comment:
```suggestion
  // You can also define args at the component level. They will apply to all the component's stories unless you overwrite them. To do so, use the args key on the default CSF export.
```suggestion
  // You can also define args at the component level; they will apply to all the component's stories unless you overwrite them. To do so, use the args key on the default CSF export:
```
```